### PR TITLE
Update stale.yml

### PR DESCRIPTION
**Pull request by ndurell**
# Description

# Check List

- [ ] Are you changing anything with the public API?
- [ ] Have you tested this change on real device?
- [ ] Are your changes backwards compatible with previous SDK Versions?
- [ ] Have you added unit test coverage for your changes?
- [ ] Have you verified that your changes are compitable with all the operating system version this SDK currently supports?

# Manual Test Plan

1.

# Supporting Materials

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   stale:
 
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```
Comment (evan-masseau) [inline .github/workflows/stale.yml:11]: and you can also just do ubuntu-latest

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -8,10 +8,10 @@ on:
 jobs:
   stale:
 
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/stale@v3
+    - uses: actions/stale@v6
```
Comment (evan-masseau) [inline .github/workflows/stale.yml:14]: is there a 6?

Comment (evan-masseau) [reply .github/workflows/stale.yml:14]: or did I write 6 before? I am on v5 anyway

Comment (ndurell) [reply .github/workflows/stale.yml:14]: 🤦🏻

**Review by evan-masseau — APPROVED** [review]

**Pull request merged**